### PR TITLE
Remove issue and pull request assignment files

### PR DESCRIPTION
**Pull request by konrad-jamrozik**
This is a PR made by the Azure SDK Engineering System team

Contributes to:
- https://github.com/Azure/azure-sdk-tools/issues/7560

### CODE REVIEW

File: .github/pull_request_assignment.yml [hunk 1]
```diff
@@ -1,100 +0,0 @@
...
-    reviewers:
-      - kristapratico
-      
-- rule:
-    # OpenAI data-plane PR
-    paths:
-      - "specification/cognitiveservices/data-plane/AzureOpenAI/**"
-      - "specification/cognitiveservices/OpenAI.Inference/**"
-    reviewers:
-      - lmazuel
```
Comment (konrad-jamrozik) [inline .github/pull_request_assignment.yml:15]: @lmazuel FYI

File: .github/pull_request_assignment.yml [hunk 2]
```diff
@@ -1,100 +0,0 @@
...
-      - "specification/confidentialledger/data-plane/**"
-    reviewers:
-      - christothes 
-
-- rule:
-    # eventgrid data-plane PR
-    paths:
-      - "specification/eventgrid/data-plane/**"
-    reviewers:
-      - JoshLove-msft
```
Comment (konrad-jamrozik) [inline .github/pull_request_assignment.yml:43]: @JoshLove-msft FYI

File: .github/pull_request_assignment.yml [hunk 3]
```diff
@@ -1,100 +0,0 @@
...
-      
-- rule:
-    # storage data-plane PR
-    paths:
-      - "specification/storage/data-plane/**"
-    reviewers:
-      - tg-msft
```
Comment (konrad-jamrozik) [inline .github/pull_request_assignment.yml:50]: @tg-msft FYI

File: .github/pull_request_assignment.yml [hunk 4]
```diff
@@ -1,100 +0,0 @@
...
-      - "specification/synapse/data-plane/**"
-    reviewers:
-      - lmazuel
-
-- rule:
-    # data-plane PR
-    paths:
-      - "specification/**/data-plane/iot**"
-    reviewers:
-      - drwill-ms
```
Comment (konrad-jamrozik) [inline .github/pull_request_assignment.yml:64]: @drwill-ms FYI

File: .github/pull_request_assignment.yml [hunk 5]
```diff
@@ -1,100 +0,0 @@
...
-
-- rule:
-    # data-plane PR
-    paths:
-      - "specification/**/data-plane/**"
-      - "dev/**/data-plane/**"
-    reviewers:
-      - anuchandy
```
Comment (konrad-jamrozik) [inline .github/pull_request_assignment.yml:72]: @anuchandy FYI

File: .github/pull_request_assignment.yml [hunk 6]
```diff
@@ -1,100 +0,0 @@
...
-      - jhendrixMSFT
```
Comment (konrad-jamrozik) [inline .github/pull_request_assignment.yml:73]: @jhendrixMSFT FYI

File: .github/pull_request_assignment.yml [hunk 7]
```diff
@@ -1,100 +0,0 @@
...
-      - tjprescott
```
Comment (konrad-jamrozik) [inline .github/pull_request_assignment.yml:74]: @tjprescott FYI

File: .github/pull_request_assignment.yml [hunk 8]
```diff
@@ -1,100 +0,0 @@
...
-    reviewers:
-        - anuchandy
-        - jhendrixMSFT
-        - tjprescott
-
-- rule:
-    paths:
-      - "documentation/**"
-    reviewers:
-      - josefree
```
Comment (konrad-jamrozik) [inline .github/pull_request_assignment.yml:89]: @josefree FYI

File: .github/pull_request_assignment.yml [hunk 9]
```diff
@@ -1,100 +0,0 @@
...
-      - weshaggard
```
Comment (konrad-jamrozik) [inline .github/pull_request_assignment.yml:90]: @weshaggard FYI

**Comment by weshaggard** [comment]
If anyone is interested in staying on top pull requests in a given area please make sure you are in the CODEOWNERS file so that you will still be added as a reviewer.

File: .github/pull_request_assignment.yml [hunk 10]
```diff
@@ -1,100 +0,0 @@
...
-    reviewers:
-      - lmazuel
-
-- rule:
-    # Cognitive Services - Language pillar data-plane PR
-    paths:
-      - "specification/cognitiveservices/data-plane/Language/**"
-    reviewers:
-      - kristapratico
-      - heaths
```
[the remark this answers is not in the document]
Comment (heaths) [reply .github/pull_request_assignment.yml:23]: No issue here. Already am (or was?) a CODEOWNER.

**Review by heaths — APPROVED** [review]

**Pull request merged**